apps/src/util/unsupportedBrowserWarning.js

```diff
@@ -0,0 +1,38 @@
+/* global appOptions */
+import {isUnsupportedBrowser, isIE11, isMobileDevice, isStorageAvailable} from '@cdo/apps/util/browser-detector';
+
+export var checkForUnsupportedBrowsersOnLoad = function () {
+  $(document).ready(function () {
+    var textDivId = null;
+    var dismissable = false;
```

Comment:
realize you're just moving code, but could replace `var`s with `let`/`const` while you're here :)

also this file should need to import $ right (would have thought you get a lint error for not doing so)